Album/Helper/ColorConvertHelper.h

```diff
@@ -0,0 +1,21 @@
+//
+//  ColorConvertHelper.h
+//  Album
+//
+//  Created by emersonmalca on 5/28/18.
+//  Copyright © 2018 Emerson Malca. All rights reserved.
+//
+#import <UIKit/UIKit.h>
+
+@interface ColorConvertHelper : NSObject
+
+// Creates a square that shows which color has been selected
+- (UIColor *)colorFromHexString:(NSString *)hexString;
+
+// Creates color based on hex string color code
+- (NSString *)hexStringForColor:(UIColor *)color;
+
+// Creates hex string based on color
+- (UIImage *)createImageWithColor:(UIColor *)color;
```

Comment:
Are comments here accurate?
- Line 18 says ` Creates hex string based on color` but the method doesn't return a string
- Line 15 says `Creates color based on hex string color code` but the method does just the opposite, it returns a string given a color

Reply:
I don't know why that got mixed up haha. Fixed!